__tests__/getTestRule.test.mjs

```diff
@@ -133,3 +133,33 @@ testRule({
 		},
 	],
 });
+
+testRule({
+	ruleName: 'at-rule-descriptor-no-unknown',
```

Comment:
`getLexer` is internal, so it can't be tested directly from a plugin.
https://github.com/stylelint/stylelint/blob/ea401b0d7852dcb0298e19537b5e82d928d6107e/lib/rules/at-rule-descriptor-no-unknown/index.cjs#L53

Therefore, the test added an existing rule that utilizes `languageOptions`.
See: [at-rule-descriptor-no-unknown | Stylelint](https://stylelint.io/user-guide/rules/at-rule-descriptor-no-unknown/)